docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc

```diff
@@ -9,84 +9,105 @@ include::_attributes.adoc[]
 :diataxis-type: tutorial
 :categories: security
 
-Here, you use the Quarkus OpenID Connect (OIDC) extension to secure a Jakarta REST application using Bearer token authentication.
+Use the Quarkus OpenID Connect (OIDC) extension to secure a Jakarta REST application with Bearer token authentication.
 The bearer tokens are issued by OIDC and OAuth 2.0 compliant authorization servers, such as link:https://www.keycloak.org[Keycloak].
 
-To better understand OIDC Bearer token authentication, see xref:security-oidc-bearer-token-authentication.adoc[OIDC Bearer token authentication].
+For more information about OIDC Bearer token authentication, see the Quarkus xref:security-oidc-bearer-token-authentication.adoc[OpenID Connect (OIDC) Bearer token authentication] guide.
 
-If you want to protect web applications by using OIDC Authorization Code Flow authentication, see xref:security-oidc-code-flow-authentication-concept.adoc[OIDC authorization code flow authentication].
+If you want to protect web applications by using OIDC Authorization Code Flow authentication, see the xref:security-oidc-code-flow-authentication.adoc[OpenID Connect authorization code flow mechanism for protecting web applications] guide.
 
 == Prerequisites
 
 :prerequisites-docker:
 include::{includes}/prerequisites.adoc[]
-* https://stedolan.github.io/jq/[jq tool]
+* The https://stedolan.github.io/jq/[jq command-line processor tool]
 
 == Architecture
 
-In this example, we build a simple microservice which offers two endpoints:
+This example shows how you can build a simple microservice that offers two endpoints:
 
 * `/api/users/me`
 * `/api/admin`
 
-These endpoints are protected and can only be accessed if a client is sending a bearer token along with the request, which must be valid (e.g.: signature, expiration and audience) and trusted by the microservice.
+These endpoints are protected and can only be accessed if a client sends a bearer token along with the request, which must be valid (for example, signature, expiration, and audience) and trusted by the microservice.
 
-The bearer token is issued by a Keycloak Server and represents the subject to which the token was issued for. For being an OAuth 2.0 Authorization Server, the token also references the client acting on behalf of the user.
+The bearer token is issued by a Keycloak server and represents the subject for which the token was issued.
```

Comment:
```suggestion
A Keycloak server issues the bearer token and represents the subject for which the token was issued.
```

Reply:
@MichalMaler - This suggestion changes the technical meaning. 

If the _Keycloak server represents the subject for which the token was issued_ and not the _bearer token_, then I will commit this change.

@sberyozkin - WDYT?

Reply:
@michelle-purcell Good catch, yes, Michal's suggestion changes the technical meaning